packages/suite/src/reducers/suite/metadataReducer.ts

```diff
@@ -41,7 +42,9 @@ const metadataReducer = (state = initialState, action: Action): MetadataState =>
                 draft.providers.push(action.payload);
                 break;
             case METADATA.REMOVE_PROVIDER:
-                draft.providers = draft.providers.filter(p => p.type !== action.payload.type);
+                draft.providers = draft.providers.filter(
+                    p => p.clientId !== action.payload.clientId,
```

Comment:
note to myself - think about google provider that can have 2 client ids

Reply:
yes, but I hope that there can't be 2 google providers connected at once at the moment. This is supported so far only on redux level, in general, on many places in metadata I now rely on the assuption that only one provider can be connected.

you can see code like this 
```
      dispatch(
                    setMetadata({
                        ...nextKeys,
                        data: nextData,
                        // todo: huh huh
                        provider: getState().metadata.providers[0]!,
                    }),
                );
```

when we are about to implement simultaneous connections to multiple providers, these places will need to be fixed